app/src/main/AndroidManifest.xml

```diff
@@ -2,14 +2,18 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.rent.application1">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
+        android:name=".MyApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="PicturesApi"
```

Comment:
Nazwę sugeruję zostawić w pliku `strings` (w xml, tak jak to było) i tam ją po prostu zmienić